### ISSUE TITLE
Taking a lot of time, to generate complete report.

### ISSUE THREAD

**Issue by saurabhvaj** [opstree-aws-utils/TagsValidator#1]
This job is taking a lot of time, as this iterate over multiple services like lambda, EC2 and S3. I request for a feature, where we can select the resources for which I can run this job. 
Or separate jobs per aws service and a complete wrapper pipeline over them.

**Comment by HarjeetPahuja** [comment]
Yes because, we are handling all api's in single call and S3 api's take to much to respond so we can separate api's call so for that we can use build promotional status plugin or can use choice parameter depends on your use case.
